### DISCUSSION TITLE
Allowing items to contain other items in the item tree

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1143] [Ideas]
I have a character, who's got an employee. I'd like to store the employee character as a sub-item under the employer, this helps with the visual representation of the organization in the item tree.

This could also be helpful with all other types of items.

The idea is to allow items to *also* behave like folders.

**Comment by vkbo** [comment]
This is already described in #1002, and implemented in the [1.7 beta](https://github.com/vkbo/novelWriter/releases/tag/v1.7b1).
